vuepress/docs/next/docs/reference/bundle-upgrade.md

```diff
@@ -0,0 +1,136 @@
+---
+sidebarDepth: 2
+---
+
+# Bundles Upgrade Guideline
+
+Two types of bundles can be utilized on Entando 7.1+. The original git-based bundle added a composable method with reusable components. The newly introduced docker-based bundles optimize their structure and expands functionality. The [Bundle Evolution Comparison](./bundle-comparison.md) table lists the differences between the two types. The following page describes the guidelines for upgrading to docker-based v5 bundles.
+
+Git-based bundles do not need to be to upgraded. Both types of bundles are compatible on Entando 7.1+, but docker-based bundles provide several innovations and an improved developer experience.
+
+The improvements include:
+* Simplified bundle directory structure
+* A decoupled build and deployment process that is easier to understand, modify, and troubleshoot
+* CI/CD integration and support 
+* One Git source repository required with Docker image repositories created automatically as needed
+* Optimized file storage and transfer methods using a layered file system, beneficial for large projects
+* Standardized bundle address format
+
+## Quick Reference
+* [Bundle Specifications](../curate/bundle-details.md)
+
+* [Bundle Management](../getting-started/ent-bundle.md)
+
+* [Bundle Version Comparison Table](./bundle-comparison.md)
+
+## Upgrade Process
+
+### Initialize Bundle 
+
+1. Initialize a new bundle to set up the standard Entando Bundle structure using the ent CLI:
+
+``` 
+ent bundle init YOUR-BUNDLE-NAME
+```
+### Copy Assets
+1. Use the ent CLI bundler command to export assets from an existing Entando Bundle. See the step-by-step [Export and Publish Tutorial](../../tutorials/create/pb/export-bundle-from-application.md). 
+
+2. `YOUR-BUNDLE-NAME/platform` directory is designated for resources such as fragments, contents, pages, and templates. Copy the output of the ent bundler to this folder. You can also accomplish this manually but there are advantages to the bundler such as generating top level descriptors for platform resources.
+Note that widgets that are not MFEs are considered platform entities on Entando and should be placed in the `platform/widgets` directory.
+
+### Add Micro Frontends & Microservices
+ 
+1. Add your micro frontend to the new bundle descriptor `entando.json`:
+``` sh
+ent bundle mfe add YOUR-MFE1
+```
+The docker-based bundle has a centralized bundle descriptor with bundle entities such as micro frontends, microservices, APIs, DBMS, and ID management specifications. 
+
+2. Copy the MFE files into `YOUR-BUNDLE-NAME/microfrontends` directory. For instance, for an MFE named YOUR-MFE1: 
+
+``` sh
+mv YOUR-ORIGIN-DIRECTORY/YOUR-MFE1/{.,}* microfrontends/YOUR-MFE1
+```
+Micro frontends and microservices are partitioned into their own directories in the docker-based bundles to streamline the CI/CD process.
```

Comment:
would reorder sentence to lead with subject or action to set the reader up to quickly sort out what's happening where: 

"In docker-based bundles, micro frontends and microservices are partitioned into their own directories to streamline the CI/CD process."
OR
"To streamline the CI/CD process, docker-based bundles partition micro frontends and microservices into their own directories."

also, no **the** before "docker-based bundles" because this is a general comment and "the" implies specificity

Reply:
? i don't understand.. this applies to all docker-based bundles used in this context

Reply:
your comment about 'the' is correct. The reordering is what I meant reads about the same